src/components/User/UserInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = (props) => {
+  const { user } = props;
+  const { name, username, email } = user;
+
```

Comment:
Here you may add one variable :
```suggestion
export const UserInfo: React.FC<Props> = (props) => {
  const { name, username, email } = props.user;

```

Reply:
ok